### PR TITLE
fix(build): some fixes for better hosting experience

### PR DESCRIPTION
**Pull request by Epicus7**
Hello there! We are running the Russian server with fork of Beestation and encountered some problems, which are pretty annoying for hosting. Here are our fixes for them.

* Add configs to .gitignore, similary to Bay builds. This fix doesn't touch subfolders of `config/`, only actual config files.
* Fix byond-extools library loading for Linux
* Use config instead of hardcoded urls in the server description for byond hub

### CODE REVIEW

**Comment by ghost** [comment]
thank you tovarish

**Comment by PowerfulBacon** [comment]
The configs are included in the repo, so that anyone that downloads a fork of Bee can have the exact config we use for a quicker setup and is manually updated

**Comment by Epicus7** [comment]
> The configs are included in the repo, so that anyone that downloads a fork of Bee can have the exact config we use for a quicker setup and is manually updated

But I can't change configs as I need without copying entire build somewhere else from the local repository. It affects git's status and causes problems with updating the repository on host.

Baystation-like builds use different folder `config/example` to store examples. There I can copy the examples to `config/`, change it as i need and everything works alright. With BeeStation i need somehow circumvent git in order to change configs and do not broke repository.

The only thing I don't sure if `config/Sage` folder can be used as example. If it doesn't and it is just a config of second server, i can move deleted configs to new `example` folder.

**Comment by francinum** [comment]
@Epicus7 You should be running your server through https://github.com/tgstation/tgstation-server, which includes support for static files, as well as various other support systems, and is tightly integrated with the codebase.

Your changes to extools loading and the hub contact fix is greatly appreciated, however.

File: .gitignore [hunk 1]
```diff
@@ -216,3 +216,9 @@ tools/MapAtmosFixer/MapAtmosFixer/bin/*
 !/config/jukebox_music/sounds/exclude
 /config/title_music/sounds/*
 !/config/title_music/sounds/exclude
+
+config/*
+!config/jukebox_music/
+!config/Sage/
+!config/title_music/
+!config/title_screens/
```
Comment (Crossedfall) [inline .gitignore:224]:
```suggestion
```

This is intentionally not ignored.

**Review by Crossedfall — CHANGES_REQUESTED** [review]
Outside of the changes to `.gitignore` I consider the rest to be a welcome cleanup.

**Comment by Crossedfall** [comment]
I actually misread the PR during my last review. I thought you were removing some trash files we used to have, but that's not actually the case. Just going to close this since it's not something we're going to be merging.

**Pull request closed without merging**